config/packages/api_platform.yaml

```diff
@@ -0,0 +1,21 @@
+services:
+    _defaults:
+        public: false
+        autoconfigure: true
+
+    api_name_converter:
+        class: Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter
+    App\Api\DataProvider\:
+        resource: '../../src/Api/DataProvider'
+
+api_platform:
+    title: MsgPHP demo API
+    version: 0.1.0
+    name_converter: api_name_converter
+    path_segment_name_generator: api_platform.path_segment_name_generator.dash
+    resource_class_directories:
+        - '%kernel.project_dir%/src/Api/Resource'
+    graphql:
+        enabled: false
+        graphiql:
```

Comment:
Isn't it weird to enable `graphiql` but not GraphQL (I guess it will just fail) :)?

Reply:
mistake from me :)